go-packages/meep-sessions/session-store.go

```diff
@@ -209,6 +213,7 @@ func getUserEntryHandler(key string, fields map[string]string, userData interfac
 		s.Sandbox = fields[ValSandbox]
 		s.Role = fields[ValRole]
 		s.Timestamp, _ = time.Parse(time.RFC3339, fields[ValTimestamp])
+		s.StartTime, _ = time.Parse(time.RFC3339, fields[ValStartTime])
```

Comment:
Do we have a metric on the Sandbox creation duration?

Reply:
Currently not. This is a simple metric to add if we need it.